sc-platform-api/src/documentation/user/index.ts

```diff
@@ -0,0 +1,5 @@
+import { UserDocumentationGetUserById } from './getUserById';
+
+export const UserDocumentation = {
+  GET_USER_BY_ID: UserDocumentationGetUserById,
```

Comment:
Name of the documentation object must be the same as method's name in controller
```suggestion
  GET_USER: UserDocumentationGetUser,
```